test/flake.nix

```diff
@@ -51,6 +51,10 @@
             "${inputs.services-flake}/nix/services/tika_test.nix"
             "${inputs.services-flake}/nix/services/weaviate_test.nix"
             "${inputs.services-flake}/nix/services/zookeeper_test.nix"
+          ] ++ lib.optionals (!pkgs.python3.pkgs.jaraco-path.meta.broken) [
```

Comment:
Mmm, I wish that the Mongo package itself had a better value for its `broken` tag, because I would rather key off of that. In the absence of it, I have moved to just checking for `isLinux`.